File: src/de/saschahlusiak/frupic/about/AboutActivity.java
package de.saschahlusiak.frupic.about;

import de.saschahlusiak.frupic.R;
import android.app.Activity;
import android.content.res.Configuration;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;
import android.view.ViewGroup.LayoutParams;
import android.widget.Button;

public class AboutActivity extends Activity {
	
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		
		setContentView(R.layout.about_activity);
		if ((getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK) != Configuration.SCREENLAYOUT_SIZE_XLARGE)
		{
			LayoutParams params = getWindow().getAttributes();
			params.width = LayoutParams.MATCH_PARENT;
			getWindow().setAttributes((android.view.WindowManager.LayoutParams) params);
		}
		
		((Button)findViewById(R.id.ok)).setOnClickListener(new OnClickListener() {
			
			@Override
			public void onClick(View v) {
				finish();		
			}
		});
	}
}

File: app/src/main/java/de/saschahlusiak/frupic/about/AboutActivity.kt
package de.saschahlusiak.frupic.about

import android.content.res.Configuration
import android.os.Bundle
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity
import de.saschahlusiak.frupic.BuildConfig
import de.saschahlusiak.frupic.R
import kotlinx.android.synthetic.main.about_activity.*

class AboutActivity : AppCompatActivity(R.layout.about_activity) {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        if (resources.configuration.screenLayout and Configuration.SCREENLAYOUT_SIZE_MASK != Configuration.SCREENLAYOUT_SIZE_XLARGE) {
            val params = window.attributes
            params.width = ViewGroup.LayoutParams.MATCH_PARENT
            window.attributes = params
        }

        version.text = BuildConfig.VERSION_NAME
        ok.setOnClickListener { finish() }
    }
}